white_label/sig/white_label/types.rbs

```diff
@@ -10,24 +10,20 @@
 module WhiteLabel
   module Types
 
+    ClientError: untyped
```

Comment:
Did ordering change?

Reply:
Yeah, it looks like the transform results in some order changes. But I dont think those are significant.  For many generators we do a sort by name.